os/fs/procfs/fs_procfsproc.c

```diff
@@ -98,6 +98,8 @@
  */
 
 #define STATUS_LINELEN 128
+#define TASH_TASK_NAME "tash"
+#define TASH_TASK_NAME_LENGTH strlen(TASH_TASK_NAME)
```

Comment:
Let's use 4 instead of strlen. But need to leave notice to modify this together with NAME string when name is changed.